packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -239,11 +240,16 @@ class AndroidCameraCameraX extends CameraPlatform {
 
   /// The camera sensor orientation.
   @visibleForTesting
-  late int sensorOrientation;
+  late double sensorOrientationDegrees;
```

Comment:
Can you expand on the dart doc here that sensor orientation degrees can change when the camera being used is changed but is independent of device orientation or app orientation. (assuming that is true, I think so based on reading this pr. )